lib/mixins/dropdown.js

```diff
@@ -81,6 +80,9 @@ export default {
                         el.addEventListener('mouseover', this.noop);
                     });
                 }
+
+                document.addEventListener('keydown', this.keydownListener);
```

Comment:
WAI-ARIA best-practices specs state that first element *should* be focused when a menu is opened